src/tools/management/commands/process_summaries.py

```diff
@@ -0,0 +1,58 @@
+import os
+from concurrent.futures import ThreadPoolExecutor
+from time import sleep
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+from api.models import SummaryCacheQueue
+from api.utils.summary_cache import update_summary_cache
+
+
+class Command(BaseCommand):
+    LOCK_ID = 8080
+    WAIT_SECONDS = 5
+
+    def acquire_lock(self):
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT pg_try_advisory_lock(%s);", [self.LOCK_ID])
+            locked = cursor.fetchone()[0]
+        return locked
+
+    def release_lock(self):
+        with connection.cursor() as cursor:
+            cursor.execute("SELECT pg_advisory_unlock(%s);", [self.LOCK_ID])
+
+    def _process_tasks(self, task):
+        try:
+            task.processing = True
+            task.save()
+            update_summary_cache(task.project_id)
```

Comment:
Do we need to pass through the test project arg?

Reply:
no because that's handled when the task gets added to the table